cypress-custom/integration/fee.test.ts

```diff
@@ -1,16 +1,21 @@
 import { ChainId, WETH } from '@uniswap/sdk'
 import { FeeInformation, FeeInformationObject } from '../../src/custom/state/fee/reducer'
+import { OrderKind } from '@gnosis.pm/gp-v2-contracts'
 
 const DAI = '0xc7AD46e0b8a400Bb3C915120d284AafbA8fc4735'
 const RINKEBY = ChainId.RINKEBY.toString()
-const FEE_QUERY = `https://protocol-rinkeby.dev.gnosisdev.com/api/v1/tokens/${WETH[4].address}/fee`
 const FEE_QUOTES_LOCAL_STORAGE_KEY = 'redux_localstorage_simple_fee'
 const FOUR_HOURS = 3600 * 4 * 1000
 const DEFAULT_SELL_TOKEN = WETH[ChainId.RINKEBY].address
 
-function _assertFeeData(fee: FeeInformation): void {
-  expect(fee).to.have.property('minimalFee')
-  expect(fee).to.have.property('feeRatio')
+const getFeeQuery = ({ sellToken, buyToken, amount, kind }: any) =>
```

Comment:
Nitpick: why not typed?

Reply:
pure laziness, it's pretty straight forward implementation but i can change this